docs/pipeline/sarek/munin.md

```diff
@@ -0,0 +1,22 @@
+# nf-core/configs: MUNIN sarek specific configuration
+
+Extra specific configuration for sarek pipeline
+
+## Usage
+
+To use, run the pipeline with `-profile munin`.
+
+This will download and launch the sarek specific [`munin.config`](../conf/pipeline/sarek/munin.config) which has been pre-configured with a setup suitable for the MUNIN cluster.
+
+Example: `nextflow run nf-core/sarek -profile munin`
```

Comment:
Is it worth mentioning `--sentieon` here and how to use it?

Reply:
No, that will be done on sarek